### ISSUE TITLE
Crash when running vignette

### ISSUE THREAD

**Issue by useobjc** [BradLarson/GPUImage3#74]
Reproduce step:
1. Run FilterShowcase example
2. Select Vignette filter

Then, app will crash with next exception:
````
validateFunctionArguments:3485: failed assertion `Fragment Function(vignetteFragment): 
argument uniform[0] from buffer(1) with offset(0) and length(40) has space for 40 bytes, but argument has a length(48).'
````

**Comment by tianye2856** [comment]
I have the same problem.

**Comment by agorskih** [comment]
Hi. There is some problem with sending VignetteUniform to the Vignette shader. To quick fix and run sample you can change [Vignette.metal](https://github.com/BradLarson/GPUImage3/blob/master/framework/Source/Operations/Vignette.metal) file.
```swift
typedef struct {
    float3 vignetteCenter;
    float3 vignetteColor;
    float vignetteStart;
    float vignetteEnd;
    float2 junk_padding; # <-- Add this line
} VignetteUniform;
```

**Comment by BradLarson** [comment]
This should be fixed as of #91
